sdk/storage/azure-storage-blob/azure/storage/blob/_download.py

```diff
@@ -205,17 +204,17 @@ def _download_chunk(self, chunk_start, chunk_end):
                         download_stream_current=self.progress_total,
                         **self.request_options
                     )
-                except HttpResponseError as error:
-                    process_storage_error(error)
-
-                try:
-                    chunk_data = process_content(response, offset[0], offset[1], self.encryption_options)
```

Comment:
I think this bit of code is trying to catch exceptions generated by `self.client.download` (few lines above this).

Reply:
same idea below